test/node-fetch/external-encoding.js

```diff
@@ -0,0 +1,43 @@
+// const chai = require('chai')
```

Comment:
Remove this file?

Reply:
Why? We want to support this eventually.